addons/website_blog/tests/test_ui.py

```diff
@@ -50,3 +50,20 @@ def test_autocomplete_with_date(self):
         self.env.ref('website_blog.opt_blog_sidebar_show').active = True
         self.env.ref('website_blog.opt_sidebar_blog_index_follow_us').active = False
         self.start_tour("/blog", 'blog_autocomplete_with_date')
+
+    def test_avatar_comment(self):
+        partner = self.env['res.partner'].create({
+            'name': 'Testing Partner',
+            'email': 'testing_partner@example.com',
+        })
```

Comment:
Maybe you can just add `TestWebsiteBlogCommon` in `class TestWebsiteBlogUi(odoo.tests.HttpCase):` inheritance so you can use the partner and test blog from the Common setup, without the need to create a partner here and without the need of the diff below to assign `blog_post_test` on `cls`